src/test/java/com/flightstats/hub/dao/aws/s3Verifier/VerifierRangeLookupTest.java

```diff
@@ -10,21 +11,22 @@
 import com.flightstats.hub.util.StringUtils;
 import com.flightstats.hub.util.TimeUtil;
 import com.google.inject.Injector;
+import lombok.extern.slf4j.Slf4j;
 import org.joda.time.DateTime;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TestName;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import static com.flightstats.hub.dao.ChannelService.REPLICATED_LAST_UPDATED;
 import static com.flightstats.hub.dao.aws.S3Verifier.LAST_SINGLE_VERIFIED;
 import static org.junit.Assert.assertEquals;
 
+@Slf4j
 public class VerifierRangeLookupTest {
-    private final static Logger logger = LoggerFactory.getLogger(VerifierRangeLookupTest.class);
+
+    private static final SpokeProperty spokeProperty = new SpokeProperty(PropertyLoader.getInstance());
```

Comment:
looks like it might be possible to declare this in the setupClass method, but I might be missing something.